bravado/http_future.py

```diff
@@ -127,7 +143,21 @@ def __init__(self, future, response_adapter, operation=None,
             also_return_response_default=False,
         )
 
-    def response(self, timeout=None, fallback_result=SENTINEL, exceptions_to_catch=FALLBACK_EXCEPTIONS):
+    @property
+    def _bravado_config(self):
+        # type: () -> BravadoConfig
+        if self.operation:
+            return self.operation.swagger_spec.config['bravado']
+        else:
+            return BravadoConfig.from_config_dict(CONFIG_DEFAULTS)
+
+    def response(
+        self,
+        timeout=None,  # type: typing.Optional[float]
+        fallback_result=SENTINEL,  # type: typing.Optional[typing.Union[typing.Any, typing.Callable[[BaseException], typing.Any]]]  # noqa
```

Comment:
This is already typed correctly internally - any reason you went with `typing.Any` instead of doing what's done there (i.e. using T)?

Reply:
I used copy/paste from docstring, but I need to check more as is reasonable to assume that `typing.Any` should be replaced with `T` as we should ensure that the `fallback_result` is of the _same type_ of the endpoint response